sdk/identity/pom.xml

```diff
@@ -16,7 +16,6 @@
 
   <modules>
     <module>azure-identity</module>
-    <module>azure-identity-v2</module>
```

Comment:
`ci.v2.yml` should be removed and `ci.yml` should be cleaned up as well in the `sdk/identity` directory.